include/maidsafe/routing/routing_node.h

```diff
@@ -80,21 +80,16 @@ class RoutingNode {
   }
 
   void AddContact(asio::ip::udp::endpoint endpoint) {
-    crux_asio_service_.service().post([=]() {
-      connection_manager_.AddNode(boost::none, EndpointPair(endpoint));
-      });
+    crux_asio_service_.service().post(
```

Comment:
I guess crux_asio_service_ can now be moved to connections class ? (after Peter's changes)